### PR TITLE
DM-11813:   Write up summary of current status and issues with A&L Decorrelation and Zogy

### PR DESCRIPTION
**Pull request by djreiss**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,61 +1,643 @@
...
+   Subsections of a DECam Zogy image subtraction, including warped and
+   PSF-matched template, science image, and the results of the
+   "standard" and image-space versions of the Zogy algorithm.
+
+.. figure:: _static/fig04b.png
+   :name: figure-4b
+
+   Subsections of the same DECam Zogy image subtraction as in Figure 4a.
+
+I should note that this fringing was observed by Tim Axelrod in another
```
Comment (ebellm) [inline index.rst]: Could you give some more detail here?  What other ZOGY implementation?  On what data?

File: index.rst [hunk 2]
```diff
@@ -1,61 +1,643 @@
...
+accounts for masks and propagates the convolutions through to the
+variance and mask planes of the exposures. The image-space convolution
+Zogy images are shown in the bottom-right of `Figure 4 <#figure-4>`__
+and `Figure 5 <#figure-5>`__. Because the convolution kernel is
+spatially constrained, we see that the artifacts evident in the
+"standard" Zogy implementation (bottom left of those figures) are also
+spatially constrained. However, it is also evident that echo-like
+artifacts are also generated in the image-space version which can be
+severe surrounding the brightest stars. These artifacts lead to a
+greater number of false positive detections (472 vs. 257 before merging;
```
Comment (ebellm) [inline index.rst]: what does "merging" refer to here?

**Review by ebellm — COMMENTED** [review]
I'd add a brief introductory paragraph explaining the goals of the DMTN (to provide an overview of the now-implemented modes of diffim and a snapshot of status and open issues).

File: index.rst (hunk 1 above)
Comment (djreiss) [reply index.rst]: Sorry for the omission. I include it in the document; it is this implementation: https://github.com/pmvreeswijk/ZOGY, and he was running it on DECam data. That is all I know (Tim suggested that his notes on this weren't too complete but might be worth following up with him -- apparently he figured out how to fix it by changing the way PSFEx computes its PSFs).

File: index.rst (hunk 2 above)
Comment (djreiss) [reply index.rst]: positive and negative sources in the diffim are merged (into dipoles etc). I'll clarify in the text.

**Pull request merged**